charts/alfresco-connector-ms365/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v2
+name: alfresco-connector-ms365
+description: A Helm chart for deploying Alfresco connector ms365 service
+type: application
+version: 0.1.0
```

Comment:
I would use a pre-release as I expect further changes might be needed once we try to integrate with acs-deployment as a parent chart